### ISSUE TITLE
[Task] Add bin/console task for Symfony

### ISSUE THREAD

**Issue by veewee** [phpro/grumphp#1126]
| Q               | A
| --------------- | ---
| Version         | `next`
| Bug?            | no
| New feature?    | yes
| Question?       | no
| Documentation?  | no
| Related tickets | 

Provide a task that makes it possible to run Symfony's `bin/console` with a specific command:

```yaml
tasks:
  symfony_console:
    command: [ "lint:container", "-vvv"]
```

This will run : `"./bin/console" "lint:container" "-vvv"`

If the command fails (non-zero exit code), the grumphp task fails.

Running multiple symfony console commands (in parallel) is already covered:

```yaml
tasks:
  lint-container:
    command: [ "lint:container", "-vvv"]
    metadata:
      task: symfony_console
```

**Comment by veewee** [comment]
Closing for now, we'll continue in the PR.
